spec/lib/authorities/linked_data/config_spec.rb

```diff
@@ -148,10 +147,10 @@
           language: ['en', 'fr', 'de'],
           total_count_ldpath: "vivo:count",
           results: {
-            id_predicate: 'http://purl.org/dc/terms/identifier',
-            label_predicate: 'http://www.w3.org/2004/02/skos/core#prefLabel',
-            altlabel_predicate: 'http://www.w3.org/2004/02/skos/core#altLabel',
-            sort_predicate: 'http://www.w3.org/2004/02/skos/core#prefLabel'
+            id_ldpath: 'dcterms:identifier',
+            label_ldpath: 'skos:prefLabel',
+            altlabel_ldpath: 'skos:altLabel',
+            sort_ldpath: 'skos:prefLabel'
```

Comment:
This is the primary change for the PR.  The config specified results as ldpaths instead of predicates.